tests/test_glm_synthetic_single_neuron.py

```diff
@@ -1,12 +1,12 @@
 import jax
 import jax.numpy as jnp
 import numpy as onp
-import neurostatslib as nsl
 from neurostatslib.glm import GLM
 from neurostatslib.basis import RaisedCosineBasis
 import matplotlib.pyplot as plt
 jax.config.update("jax_platform_name", "cpu")
 jax.config.update("jax_enable_x64", True)
+import neurostatslib
```

Comment:
what's the matter with `import neurostatslib as nsl`?

Reply:
sounds good

Reply:
for now I kept it consistent with the main